packages/office-ui-fabric-react/src/components/TextField/examples/TextField.Basic.Example.tsx

```diff
@@ -3,15 +3,25 @@ import { TextField, MaskedTextField } from 'office-ui-fabric-react/lib/TextField
 import { Stack } from 'office-ui-fabric-react/lib/Stack';
 
 export const TextFieldBasicExample: React.StatelessComponent = () => {
+  // TextFields don't have to be inside Stacks, we're just using Stacks for layout
   return (
-    <Stack gap={15} maxWidth={300}>
-      <TextField label="Standard" />
-      <TextField label="Disabled" disabled />
-      <TextField label="Read-only" readOnly />
-      <TextField label="Required " required />
-      <TextField label="With error message" errorMessage="Error message" />
-      <MaskedTextField label="With input mask" mask="m\ask: (999) 999 - 9999" />
-      <TextField label="With an icon" iconProps={{ iconName: 'Calendar' }} />
+    <Stack horizontal gap="50" styles={{ root: { width: 650 } }}>
+      <Stack gap="15" styles={{ root: { width: 300 } }}>
+        <TextField label="Standard" />
+        <TextField label="Disabled" disabled defaultValue="I am disabled" />
+        <TextField label="Read-only" readOnly defaultValue="I am read-only" />
+        <TextField label="Required " required />
+        <TextField label="With error message" errorMessage="Error message" />
+      </Stack>
+
+      <Stack gap="15" styles={{ root: { width: 300 } }}>
+        <MaskedTextField label="With input mask" mask="m\ask: (999) 999 - 9999" />
+
```

Comment:
nit: Why are there blank lines separating these? (Same with other files below)

Reply:
Attempt at improving readability...not sure if it actually helps